pom.xml

```diff
@@ -0,0 +1,214 @@
+          <groups>${testcase.groups}</groups>
+        </configuration>
+      </plugin>
+      <!-- ${project.build.outputDirectory}/git.properties is generated at compile time -->
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>4.9.10</version>
+        <executions>
+          <execution>
+            <id>get-the-git-infos</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+            <phase>initialize</phase>
+          </execution>
+        </executions>
+        <configuration>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
+          <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties
+          </generateGitPropertiesFilename>
+          <includeOnlyProperties>
+            <includeOnlyProperty>^git.build.(time|version)$</includeOnlyProperty>
+            <includeOnlyProperty>^git.branch$</includeOnlyProperty>
+            <includeOnlyProperty>^git.commit.(id.full|time)$</includeOnlyProperty>
+            <includeOnlyProperty>^git.dirty$</includeOnlyProperty>
+          </includeOnlyProperties>
+          <commitIdGenerationMode>full</commitIdGenerationMode>
+          <failOnUnableToExtractRepoInfo>false</failOnUnableToExtractRepoInfo>
+          <failOnNoGitDirectory>false</failOnNoGitDirectory>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <!--system scope-->
+    <dependency>
+      <groupId>com.avispl.symphony.api</groupId>
+      <artifactId>symphony-api-commons</artifactId>
+      <version>5.3.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/symphony-api-commons-5.3.0.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.avispl.symphony.api</groupId>
+      <artifactId>symphony-api-dal</artifactId>
+      <version>5.3.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/symphony-api-dal-5.3.0.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.avispl.symphony.dal</groupId>
+      <artifactId>symphony-dal-commons</artifactId>
+      <version>5.3.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/symphony-dal-commons-5.3.0.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>org.icmp4j</groupId>
+      <artifactId>icmp4j</artifactId>
+      <systemPath>${basedir}/lib/icmp4j-1018.jar</systemPath>
+      <scope>system</scope>
+      <version>1018</version>
+    </dependency>
+    <!--provided scope-->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <scope>provided</scope>
+      <version>2.11.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <scope>provided</scope>
+      <version>2.11.1</version>
```

Comment:
please check dependencies according to the confluence doc "3rd Party Software - Symphony". This dependency is slightly outdated, maybe there's more

Reply:
I fixed it